From 5a7ba0859627a94267aed9b2d724fcb9db0d3d46 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 29 Apr 2021 22:56:15 +0300
Subject: [PATCH] [SW-40350] fwupd: print received commands

Print every FWUPD received command except for CLR_STS.
This is helpful to understand what is going on.
The CLR_STS command is not saved on the state machine and therefore is
not skipped and executed each time, so skip printing it.

Change-Id: Idfd5d54dc2fda2bec24cf92e4ce373b8c1d000d4
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot/module_fwupd.c b/preboot/module_fwupd.c
index 6dfcb0e..852e12b 100644
--- a/preboot/module_fwupd.c
+++ b/preboot/module_fwupd.c
@@ -252,9 +252,11 @@
 
 	module_fwupd_state_shift();
 
-	preboot_puts_raw_dbg("FW UPD ");
-	preboot_puts_raw_dbg(cmd_str);
-	preboot_puts_dbg(" received\n");
+	if (state.cmd.cmd != HL_UPD_CLR_STS) {
+		preboot_puts_raw("FW UPD ");
+		preboot_puts_raw(cmd_str);
+		preboot_puts(" received\n");
+	}
 }
 
 __weak void preboot_nic_en(u32 nic_en) {}
